src/analysis/retail/demonhunter/vengeance/modules/talents/FrailtyDebuff.tsx

```diff
@@ -1,38 +1,34 @@
 import { t } from '@lingui/macro';
-import { formatDuration, formatPercentage, formatThousands } from 'common/format';
+import { formatDuration, formatPercentage } from 'common/format';
 import SPELLS from 'common/SPELLS/demonhunter';
 import { TALENTS_DEMON_HUNTER } from 'common/TALENTS/demonhunter';
 import { SpellLink } from 'interface';
 import UptimeIcon from 'interface/icons/Uptime';
 import Analyzer, { Options } from 'parser/core/Analyzer';
 import { NumberThreshold, ThresholdStyle, When } from 'parser/core/ParseResults';
-import AbilityTracker from 'parser/shared/modules/AbilityTracker';
 import Enemies from 'parser/shared/modules/Enemies';
 import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
 import Statistic from 'parser/ui/Statistic';
 import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
 import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 
-class SpiritBombFrailtyDebuff extends Analyzer {
+class FrailtyDebuff extends Analyzer {
   static dependencies = {
-    abilityTracker: AbilityTracker,
     enemies: Enemies,
   };
 
-  protected abilityTracker!: AbilityTracker;
   protected enemies!: Enemies;
 
   constructor(options: Options) {
     super(options);
-    this.active = this.selectedCombatant.hasTalent(
-      TALENTS_DEMON_HUNTER.SPIRIT_BOMB_VENGEANCE_TALENT.id,
-    );
+    this.active =
+      this.selectedCombatant.hasTalent(TALENTS_DEMON_HUNTER.SPIRIT_BOMB_VENGEANCE_TALENT.id) ||
+      this.selectedCombatant.hasTalent(TALENTS_DEMON_HUNTER.FRAILTY_VENGEANCE_TALENT.id) ||
+      this.selectedCombatant.hasTalent(TALENTS_DEMON_HUNTER.VOID_REAVER_VENGEANCE_TALENT.id);
   }
```

Comment:
would be interesting to see avg DR / leech % / dmg increase later on (pending tuning) since all are supposed to stack

Reply:
I would love to include that kind of info, I'm just away for a bit and won't be able to get to it until I get back next week.